atsign-foundation/at_libraries
# fix: update exception handling for AtLookup.findSecondary()

Closes https://github.com/atsign-foundation/sshnoports/issues/483

**- What I did**
- update exception handling for AtLookup.findSecondary()

**- How I did it**
- split some very long exception messages into a log message and the rest as an exception message for more clarity
- add special exception handling for RootServerConnectivityException in CacheableSecondaryAddressFinder
- some more minor modifications to exception messages that are part of the finding secondary address process
- some minor formatting of exception messages and code comments in at_auth_impl

**- How to verify it**
- The updated exception message when the internet is disconnected should be **Unable to authenticate | Cause: Unable to establish connection with root server. Please check your internet connection and try again**

**- Description for the changelog**

fix: update exception handling for AtLookup.findSecondary()
